contracts/interfaces/IERC20Detailed.sol

```diff
@@ -0,0 +1,13 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.6.8;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface IERC20Detailed is IERC20 {
+  // ERC20 Optional Views
+  function name() external view returns (string memory);
+
```

Comment:
Extra blank lines here seem unnecessary. It's different than how the staking bank interface is formatted.

Reply:
I kept blank line (in all interfaces), for me is also looks better and I take a look on synthetics, they also using one blank line in interfaces so I think we can do the same.